netkan/netkan/github_pr.py

```diff
@@ -38,12 +38,12 @@ def create_pull_request(self, title: str, branch: str, body: str) -> None:
             except KeyError:
                 pass
             logging.error('PR for %s failed: %s - %s',
-                branch,
-                message,
-                error
-            )
+                          branch,
+                          message,
+                          error
+                          )
```

Comment:
This looks a little strange, generally would have these on one line. 
```suggestion
                          branch,  message,  error)
```
Or formatted how it was. But :man_shrugging:

Reply:
Yeah, one of the tools thinks that function parameters should be indented the same when the first one is on the same line as the open parenthesis. I have seen this work both well and poorly; will combine these onto one line.